### PR TITLE
North West Class 4 - Oneil Malik - homework.md - Week 1

### PR DESCRIPTION
**Pull request by Neil0696**
# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

### CODE REVIEW

File: homework.md [hunk 1]
```diff
@@ -44,43 +44,45 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
-A programming language used on the internet
+A programming language used to make a website interactive
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+1) When people want to show off code to each other they can put it on Github <br>
```
Comment (dharmaguadeloupe) [inline homework.md:53]: Left the `<br>` tag in

File: homework.md [hunk 2]
```diff
@@ -44,43 +44,45 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
+2) It helps to make collabration easy with others <br>
```
Comment (dharmaguadeloupe) [inline homework.md:54]: Again same issue with the `<br>` tag

**Review by dharmaguadeloupe — COMMENTED** [review]
Just a couple of erroneous tags.

**Comment by Neil0696** [comment]
@dharmaguadeloupe Oh silly me. I didn't even realise, but thanks for that!

**Pull request closed without merging**